src/pages/index.js

```diff
@@ -64,44 +110,72 @@ profileModal.setEventListeners();
 const addCardModal = new PopupWithForm({
   popupSelector: "#add-card-modal",
   handleFormSubmit: (formData) => {
-    const newCard = createCard({ name: formData.Title, link: formData.url });
-    cardSection.addItem(newCard);
-    addCardModal.close();
-    addCardFormValidator.disableButton();
+    api
```

Comment:
Every server request in `index.js` should look something like this:

```js
// show `Saving...`
api.someMethod()
.then(res => {
// here is the response that should be used to update the DOM.
// also in `then` you should close the popup
})
.catch(err => // here you catch possible errors)
.finally(() => // here you return the default button text back )
``` 

Please, correct it in the whole code